api-router.js

```diff
@@ -139,33 +165,15 @@ router.get('/rsk-activity-report/developer-activity', async (req, res) => {
     const startDate = req.query['start-date'];
     const endDate = req.query['end-date'];
 
-    const activityReport = await rskActivityReport
-      .queryDeveloperActivity(
-        startDate,
-        endDate,
-        chain,
-        windows,
-      );
-    res.status(200).json({
-      endPointVersion,
-      ...activityReport
-    });
-  } catch (error) {
-    res.status(400).json({
-      endPointVersion,
-      error: error.message,
-    });
-  }
-});
-
-
-router.get('/rsk-activity-report/avg-tx-cost', async (req, res) => {
```

Comment:
I completely removed the get route. It is no longer needed. All data specific to endpoint is specified in `endpointParameters`